compiler/test/dotty/tools/repl/LoadTests.scala

```diff
@@ -82,4 +83,13 @@ object LoadTests {
     file
   }
 
+  private val pattern = Pattern.compile("\\r[\\n]?|\\n");
+
+  // Ensure 'expected' and 'actual' contain the same line separator(s).
+  private def assertMultiLineEquals(expected: String, actual: String): Unit = {
```

Comment:
Instead of duplicating this method in two file, could you use `ReplCompilerTests.assertMultiLineEquals` in thsi file ?

Reply:
Fine for me if having a dependency between `LoadTests` and `ReplCompilerTests` is ok for you !